client/components/happychat/style.scss

```diff
@@ -159,6 +159,21 @@
 	animation: happychat-minimize-right .5s 1 forwards;
 }
 
+.layout.has-no-sidebar .happychat .happychat__container.is-open.is-minimizing {
+	animation: happychat-fade-out-right .5s 1 forwards;
```

Comment:
I don't think you need `forwards`.

Reply:
I'm temped to leave this for consistency across the file if someone else edits it later on.